src/gtest/test_mempool.cpp

```diff
@@ -136,3 +139,35 @@ TEST(Mempool, TxInputLimit) {
     EXPECT_EQ(state4.GetRejectReason(), "bad-txns-version-too-low");
 }
 
+// Valid overwinter v3 format tx gets rejected because overwinter hasn't activated yet.
+TEST(Mempool, overwinter_not_active_yet) {
+    SelectParams(CBaseChainParams::TESTNET);
```

Comment:
Non-blocking: This test will fail if Overwinter is ever set to `ALWAYS_ACTIVE` on testnet. I doubt we will do so (only case I can imagine is some custom private testnet), but downstream code forks might want to. Not too concerned however because doing this would also require changes to the genesis block.

Reply:
Added comment about this to test.